frontend/src/components/vision/sections/VisionGoals.tsx

```diff
@@ -9,22 +9,24 @@ export default function VisionGoals() {
 
   return (
     <VisionPageSection className={"bg-base-lightest"} title={t("title")}>
-      <div className="margin-top-1" />
       {contentItems.map((contentRows, contentRowsIdx) => (
-        <div className="grid-row" key={`vision-goals-${contentRowsIdx}`}>
+        <div
+          className="grid-row grid-gap"
+          key={`vision-goals-${contentRowsIdx}`}
+        >
           {contentRows.map((contentRowItem, contentRowItemIdx) => (
             <div
-              className="margin-bottom-3 tablet-lg:grid-col-6 tablet-lg:padding-right-5"
+              className="margin-bottom-4 tablet:grid-col-6"
               key={`vision-goals-${contentRowsIdx}-${contentRowItemIdx}`}
             >
-              <h3 className="font-sans-sm margin-0 tablet:font-sans-md">
+              <h3>
                 {t(`contentItems.${contentRowsIdx}.${contentRowItemIdx}.title`)}
               </h3>
-              <div className="font-sans-xs margin-top-1 line-height-sans-4">
+              <p>
                 {t(
                   `contentItems.${contentRowsIdx}.${contentRowItemIdx}.content`,
                 )}
-              </div>
+              </p>
```

Comment:
- Use grid and gap appropriately instead of manual padding classes 

(Note this could use the `Grid` component instead of manual grid clases)